README.adoc

```diff
@@ -127,12 +140,15 @@ followed by fields that correspond to metadata fields.
 *The `description` field is optional and describes the purpose of the metric
 annotation.
 
-The `@Counted` annotation applies to the `list()` method. This annotation counts how many times you request the list of systems methods by visiting `http://localhost:9080/inventory/systems` to view the current contents of the
-inventory.
-* The `absolute` field makes the name of this annotation take the name of the method.
+The `@Counted` annotation applies to the `list()` method. This annotation counts how many times
+you request the list of systems methods by visiting `http://localhost:9080/inventory/systems` to
+view the current contents of the inventory.
+* The `absolute` field gives this annotation the name of the method.
 * The `description` field is optional and describes the purpose of the metric annotation.
-* The default value of the `monotonic` field is `false`. When `false`,
-the annotation counts the current invocations of the annotated method, and the counter is incremented before the annotated method returns. However, set the value to `true` instead.
+* The `monotonic` field is set to `true` so that the counter will only increment.
+
+The `@Counted` and `@Timed` annotations can be also placed in the Resource as
+well, such as `InventoryResource`.
```

Comment:
The "also" and "as well" in the same sentence is redundant. Let's remove "as well." So we don't split the parts of the verb, let's also move "also" to the beginning:

Also, the `@Counted` and `@Timed` annotations can be placed in the Resource, such as `InventoryResource`.

Reply:
Done